### DISCUSSION TITLE
Rate change mods should affect the beatmap progress bar like it does on the beatmap wedge

### DISCUSSION THREAD

**Discussion by poiqzy** [ppy/osu#29408] [Ideas] [answered]
1 minute long beatmap:

| Area          | Displayed time |
|---------------|----------------|
| Beatmap wedge | 1 minute ✅     |
| Progress bar  | 1 minute ✅     |

1 minute long beatmap with **Half Time (0.5x) mod applied**:

| Area          | Displayed time   |
|---------------|------------------|
| Beatmap wedge | 2 minutes ✅      |
| Progress bar  | still 1 minute ❓ |

With this being the case, it is hard to know how much time is really left when having arbitrary rate change mods active.

**Comment by Tenexxt** [comment]
**This is also happening with the DT mod**. I think it was made like this on purpose, but i agree that making it count normaly with the new playtime, rather than the orginal one sped up/slowed down makes more sense (**Atleast to me**)
There is probably a reason why this wasn't added, so if anyone knows why/you're a dev, **please reply** because im kinda curious why it works the way it works.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/8929#issuecomment-623014048
